### DISCUSSION TITLE
TS2307: Cannot find module 'rsuite/esm/@types/common' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by RahulNag6896** [rsuite/rsuite#3823] [Q&A] [answered]
getting this error TS2307: Cannot find module 'rsuite/esm/@types/common' or its corresponding type declarations.
I was using rsuit v5.53.2 tried upgrading it to v5.55.0 but still getting the same error

**Comment by simonguo** [comment, marked as answer]
Try upgrading to version 5.62.1
